twilight-model/src/poll/mod.rs

```diff
@@ -0,0 +1,221 @@
+pub mod answer;
+pub mod answer_count;
+pub mod layout_type;
+pub mod media;
+pub mod results;
+
+use crate::util::Timestamp;
+use serde::{Deserialize, Serialize};
+
+pub use self::{
+    answer::PollAnswer,
+    answer_count::AnswerCount,
+    layout_type::PollLayoutType,
+    media::{PartialPollMediaEmoji, PollMedia},
+    results::PollResults,
+};
+
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct Poll {
+    /// Each of the answers available in the poll.
+    pub answers: Vec<PollAnswer>,
+    /// Whether a user can select multiple answers.
+    pub allow_multiselect: bool,
+    /// The time when the poll ends.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub expiry: Option<Timestamp>,
```

Comment:
This field is nullable and not optional
```suggestion
    pub expiry: Option<Timestamp>,
```